### DISCUSSION TITLE
無程式碼時代來了！工程師不足、數位轉型需求加劇，企業如何找到「快解方」？

### DISCUSSION THREAD

**Discussion by vvvioletta** [1101-GS4719-C/Python-Community-GS4719-C#56] [General]
前幾天在數位時代podcast聽到Low-code、No-code時代來臨，對於現在正在學python的我，無疑是一個大大的問號，因為我就是那個認為未來程式語言將會是國民基本常識的人。
後來把整集podcast聽完，又上網查了相關新聞，總結一下我的想法：
因為我是商管背景的學生，未來想走產品管理或供應鏈相關的工作，對於我而言，Low-code、No-code無疑是一大福音，因為可以使用已經開發好的平台，利用點拉拖就完成我想做到的功能（偏個人工作上的，例如專案管理、顧客關係管理等等）。可是我認為學python還是有其必要性。假如我今天是PM，工程師一直跟我說他的東西開發不出來，我有程式概念的話，也許可以跟他進行更有效率的溝通，進而讓專案更好執行——不需要原定那麼強大的功能，但可以大大縮短開發的時間，一樣達到KPI。
以上是我個人的看法，歡迎底下留言討論～
新聞來源：[https://www.bnext.com.tw/article/65287/no-code-generation-is-coming](url)

**Comment by chia-kai-chang** [comment]
@vvvioletta 不只轉貼文章，也說明了
1. 她為了這篇文章所做的努力 (聽完podcast內容)
2. 個人意見看法

我認為轉載文章應該要以這篇為範本。除了轉載文章之外，也要抒發個人意見。💯
